module3/projects/consultancy/index.md

```diff
@@ -0,0 +1,88 @@
+  * "It came up in just about every interview I had."
+  * "People seemed most excited/interested about working on a larger team project. They asked lots of followup questions about how we handled project management, made decisions, etc. It was really helpful to know how SOA/multi-app systems worked."
+  * "They were surprised that we got exposure to SOA and speaking about microservices.
+  It helped to understand and speak to Abstraction and Encapsulation a lot more. Working o
+  n a larger team definitely helped me get started at my job which was a way bigger team t
+  han the 10-person project."
+  * "People seemed most excited/interested about working on a larger team project (10 p
+  eople). They asked lots of followup questions about how we handled project management, m
+  ade decisions, etc. It was really helpful to know how SOA/multi-app systems worked."
+  * "It definitely came up in interviews, especially about microservices, and the trade
+  offs versus monolith apps."
+
+### User Empathy, Personas, and Equity Analysis
+
+We'll also examine our end users on a deeper level to instill a strong sense of developer empathy. We will do this by focusing time on "user personas" to answer questions like "who is our user", "what age range are we targeting", "how tech-savvy must the user be to use our app" and so on. As part of this project, groups will also perform an equity analysis in order to interrogate the product they're building, and reflect on possible changes that could better prioritize inclusion and harm reduction. 
+
+## Phases
+
+* [Ideation](./ideation)
+* [Inception](./inception)
+* [Development](./development)
+* [Retro](./retro_guide)
+* [Presentation](./presentation)
+* [Evaluation](./evaluation)
+
+## Technical Requirements
+
+You have a lot of freedom with this project, but there are a few technical requirements. Overall, your project must:
+
+1. Includes at least one Rails API and one ReactJS application.
+2. Use one external API, but it can be consumed on the FE or BE. 
+3. Be deployed
+4. Assume one single user - multiple users with log in and authentication is out of scope.
+5. Implement functionality beyond simply data display - there needs to be some way for the user to work with or manipulate the data (favoriting, searching, filtering, commenting, rating, etc)
+
+<section class="dropdown">
+### Back End Requirements
+
+* Thorough testing at both the unit and integration level.
+  * Test Coverage tool like SimpleCov is used
+  * Happy Path and Sad Path/Edge Case testing is included
+* One Stretch Tech option (listed below) is implemented
```

Comment:
I like this idea of limited stretch choices versus the List of Glory which, while it has excellent ideas, can feel really overwhelming. Perhaps LOG moves over to Mod 4 or is otherwise documented somewhere for post-graduation practice.